swisscom_ai/research_keyphrase/model/extractor.py

```diff
@@ -13,13 +13,13 @@
         {<NN.*|JJ>*<NN.*>}  # Adjective(s)(optional) + Noun(s)"""
 
 GRAMMAR_DE = """
-NBAR:
-        {<JJ|CARD>*<NN.*>+}  # [Adjective(s) or Article(s) or Posessive pronoun](optional) + Noun(s)
-        {<NN>+<PPOSAT><JJ|CARD>*<NN.*>+}
+         NBAR:
+         {<PPOSAT>*<PTKNEG>*<PIAT>*<ADJ.*|JJ|CARD>*<ADV>*<NN.*>+}  # [Adjective(s) or Article(s) or Posessive pronoun](optional) + Noun(s)
```

Comment:
What about modifying the comments so that they match the changes in the grammar. Otherwise I think it's a bit confusing

Reply:
you are absolutely right...I just was lazy ;)